webui/src/pages/extension-detail/extension-detail.tsx

```diff
@@ -243,10 +245,17 @@ export class ExtensionDetailComponent extends React.Component<ExtensionDetailCom
                     </Box>
                     <Box>
                         <Switch>
+                            <Route path={ExtensionDetailRoutes.CHANGES}>
+                                <ExtensionDetailChanges
+                                    extension={extension}
+                                    user={this.context.user}
```

Comment:
There's no need to pass the user. The ExtensionDetailChanges component can fetch it from the context.

Reply:
Noted. Again this is part of the `changelog sync` commit, should be resolved when commit is reverted.